src/shared/services/form/field-type-sanitiser.ts

```diff
@@ -12,11 +12,22 @@ export class FieldTypeSanitiser {
    * @param editForm
    */
    sanitiseLists(caseFields: CaseField[], editForm: any) {
-
     this.getDynamicListsFromCaseFields(caseFields).forEach(dynamicField => {
-      this.getListOfKeysFromEditForm(editForm).forEach((key) => {
-        this.createValueCodePairAlongWithListIfKeyExistsInForm(dynamicField, key, editForm);
```

Comment:
Did we remove the method as it is not used anywhere else and is not being used here now?